File: src/main/java/ar/edu/unq/epers/unidad5/Unidad5EjemploMongodbSpringApplication.java
package ar.edu.unq.epers.unidad5;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;

@SpringBootApplication
@EnableMongoRepositories
public class Unidad5EjemploMongodbSpringApplication {

    public static void main(String[] args) {
        SpringApplication.run(Unidad5EjemploMongodbSpringApplication.class, args);
    }

}

File: src/main/kotlin/ar/edu/unq/epers/unidad5/Unidad5EjemploMongodbSpringApplication.kt
package ar.edu.unq.epers.unidad5

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories

@SpringBootApplication
@EnableMongoRepositories
class Unidad5EjemploMongodbSpringApplication

fun main(args: Array<String>) {
    runApplication<Unidad5EjemploMongodbSpringApplication>(*args)
}
